dotcom-rendering/src/components/KeyTakeaway.tsx

```diff
@@ -64,7 +64,7 @@ export const KeyTakeaway = ({
 }: KeyTakeawayProps) => {
 	return (
 		<>
-			<li css={keyTakeawayStyles}>
+			<li css={keyTakeawayStyles} data-spacefinder-role="nested">
```

Comment:
Would it be possible to [document this special attribute in the JSX namespace](https://github.com/guardian/dotcom-rendering/blob/9a1870d83a92e36c2866faeaacb52a9817bdd016/dotcom-rendering/index.d.ts#L419)?